projects/valtimo/process-link/src/lib/components/form-flow/form-flow.component.ts

```diff
@@ -26,43 +26,58 @@ import {
 } from '@valtimo/components';
 import {FormFlowService} from '../../services';
 import {FormFlowInstance, FormFlowStepType} from '../../models';
+import {TranslateService} from '@ngx-translate/core';
+import {Step} from 'carbon-components-angular';
+import {ConfigService} from '@valtimo/config';
 
 @Component({
   selector: 'valtimo-form-flow',
   templateUrl: './form-flow.component.html',
-  styleUrls: ['./form-flow.component.css'],
+  styleUrls: ['./form-flow.component.scss'],
 })
-export class FormFlowComponent implements OnInit {
+export class FormFlowComponent implements OnInit, OnDestroy {
   @ViewChild('form') form: FormioComponent;
   @Input() formIoFormData: BehaviorSubject<any> | null = new BehaviorSubject<any>(null);
   @Input() formFlowInstanceId: string;
   @Output() formFlowComplete = new EventEmitter();
 
+  public readonly breadcrumbs$ = new BehaviorSubject<Step[]>([]);
   public readonly disabled$ = new BehaviorSubject<boolean>(false);
   public readonly formFlowStepType$ = new BehaviorSubject<FormFlowStepType | null>(null);
   public readonly FormFlowCustomComponentId$ = new BehaviorSubject<string>('');
+  public readonly currentStepIndex$ = new BehaviorSubject<number>(0);
+  public readonly enableFormFlowBreadcrumbs!: boolean;
```

Comment:
use observable feature toggle function so you don't have to set it in config service

Reply:
Changed